src/components/Navbar.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <nav>
+      <ul>
+        <li>
+          <Link to="/" className="cms-logo">
+            Bookstore CMS
+          </Link>
+        </li>
+        <li>
+          <Link to="/books">Books</Link>
```

Comment:
- [ ] The above-stated change, therefore, means that we have to refactor this Link too so that it can point to the home path.

  ```JavaScript
  <Link to="/">Books</Link>
  ```

Reply:
Let me rectify that. Hadn't realized it. Thank you.